building/devices/zynqmp.rst

```diff
@@ -1,150 +1,57 @@
-process, log in using username ``root`` and password ``root``. Start the OP-TEE 
-Normal World service and run xtest:
-
-.. code-block:: bash
-	
-	$ tee-supplicant -d
-	$ xtest
-	
-You can close QEMU session at any time by typing ``Ctrl-A+C`` and entering the 
-``quit`` command.
-
-Configuring and building for other ZynqMP boards
-*************************************************
-To use this makefile with other supported boards, you have to download the 
-corresponding BSP and add option ``PLATFORM`` to each make command.
-
-.. code-block:: bash
-	 
-	$ make -f zynqmp.mk PLATFORM=zcu106
-	$ make -f zynqmp.mk PLATFORM=zcu106 qemu
-
-Hereafter the list of available ``PLATFORM``:
-
-	* ``zcu102``
-	* ``zcu104``
-	* ``zcu106``
-	* ``ultra96-reva``
-
-.. warning::
-
-	On Ultra96 board, UART is not directly available. You have to connect
-	through WIFI Access Point using the procedure detailed here 
-	`Getting started`_.
-
-SD card creation
-****************
-After completion of building process, you can create a bootable SD card. Here,
-we consider that SD card corresponds to ``/dev/sdb``. We will use ``gparted``
-and ``e2image`` tools.
-
-Using ``gparted`` or any other partition manager tool create two partitions on 
-the card:
+Boot Firmware
+*****************************************
+Xilinx Zynq MPSoC device requires two firmware images, one to configure the device (First Stage Bootloader) and one for runtime platform management (PMU Firmware). The scope of the build Makefile does not cover buildling these two firmware images therefore pre built binaries are required to generate a valid boot image. The pre built images can be found in the following `Xilinx wiki <https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/18842316/Linux+Prebuilt+Images>`_ page.
 
-	* 1GB FAT32 bootable partition (``/dev/sdb1`` hereafter).
-	* EXT4 partition on the remaining memory space (``/dev/sdb2``
-	  hereafter).
+Build instructions
+*****************************************
+Follow the instructions at ":ref:`get_and_build_the_solution`" page.
 
-Once SD card is partitioned, use the following commands:
+Configuration switch ``PLATFORM`` can be used to specify the target device
+as listed in table below:
 
-.. code-block:: bash
-	 
-	$ cp /path/to/project/images/linux/BOOT.BIN /dev/sdb1
-	$ cp /path/to/project/images/linux/image.ub /dev/sdb1
-	$ sudo e2image -rap /path/to/project/images/linux/rootfs.ext4 /dev/sdb2
++------------------------+--------------------------------------+
+| Board Name             | Build configuration directive        |
++========================+======================================+
+| ZCU102                 | ``PLATFORM=zynqmp-zcu102``           |
++------------------------+--------------------------------------+
 
-Now you can use the newly created SD card to boot your board.
+After completion of the buildling process, two new files will be generated within the zynqmp folder, BOOT.bin and <platform-name>.ub.
```

Comment:
80 char/line for the text. Please wrap.

Mark file names with "``":
```patch
- within the zynqmp folder, BOOT.bin and <platform-name>.ub.
+within the ``zynqmp/`` folder: ``BOOT.bin`` and ``<platform-name>.ub``. 
```

Reply:
Is the wrapping really required? I mean, don't want to question if there is a guideline that requires it but I thought Sphinx would render the long lines appropriately. I have more lines >80 so I will wrap all of them if so

Reply:
> Is the wrapping really required?

Make it really easier for readers to follow the docs and adds consistency between articles in the docs.